components/collective-page/sections/Contribute.js

```diff
@@ -25,6 +27,12 @@ import SectionTitle from '../SectionTitle';
 
 const CONTRIBUTE_CARD_PADDING_X = [15, 18];
 
+const ContributeCardContainer = styled(Box).attrs({ px: CONTRIBUTE_CARD_PADDING_X })(
```

Comment:
I did this on purpose, they all share the same `px` in the file so it made sense to add it at the component level rather than passing it for every files. If `styled(Box)` wasn't a `Box`, using `attrs({ px: CONTRIBUTE_CARD_PADDING_X })` wouldn't work, `attrs` is just a way to set the props (and not the styles directly) for the component.